onnx/hub.py

```diff
@@ -348,3 +348,65 @@ def download_model_with_test_data(
     )
 
     return model_with_data_path
+
+
+def load_composite_model(
+    network_model: str,
+    preproc_model: str,
+    network_repo: str = "onnx/models:main",
+    preproc_repo: str = "onnx/models:main",
+    opset: Optional[int] = None,
+    force_reload: bool = False,
+    silent: bool = False,
+) -> Optional[onnx.ModelProto]:
+    """
+    Builds a composite model including data preprocessing by downloading a network and a preprocessing model
+    and combine it into a single model
+
+    :param model: The name of the onnx model in the manifest. This field is case-sensitive
+    :param repo: The location of the model repo in format "user/repo[:branch]".
+        If no branch is found will default to "main"
+    :param opset: The opset of the model to download. The default of `None` automatically chooses the largest opset
+    :param force_reload: Whether to force the model to re-download even if its already found in the cache
+    :param silent: Whether to suppress the warning message if the repo is not trusted.
+    :return: ModelProto or None
+    """
+    preproc = load(preproc_model, preproc_repo, opset, force_reload, silent)
+    network = load(network_model, network_repo, opset, force_reload, silent)
+    if preproc is None or network is None:
+        return None
```

Comment:
Perhaps throw an error saying one of them is invalid?

Reply:
Done